test/src/qa/hpcp/test_hpcp_parameters.py

```diff
@@ -0,0 +1,104 @@
+        'BPF WN midi ref. 28', 
+        'BPF WN midi ref. 29', 
+        'BPF WN midi ref. 30', 
+                 )
+# Test titles
+testTitles = {
+    'testWindowWholeRange': 'HPCP and window type in different scales',
+    'testFrameSize': 'HPCP, window type and frame size in the lower scales', 
+    'testMaxFreq': 'HPCP, window type and max frequency in the upper octaves',
+    'testJordiNL': 'Jordi non-linear filtering',
+    'testWeigth': 'Weigthing function and window type',
+    'testLPFilterMeanSpectrum': 'Mean spectrum of low pass filtered white noise',
+    'testBPFilterMeanSpectrum': 'Mean spectrum of band pass filtered white noise',
+    'testLPFilterFrameSpeaks': 'Frame spectral peaks of low pass filtered white noise',
+    'testBPFilterFrameSpeaks': 'Frame spectral peaks of band pass filtered white noise',
+    'testNormalizeWindow': 'Window normalization for different window types',
+    'testNormalizeHPCP': 'HPCP normalization',
+}
+
+# Mapping between the selected test and the probe 
+test2SignalsMapping = {'testWindowWholeRange': testSignals[0:10],
+                            'testFrameSize': testSignals[1:4], 
+                            'testMaxFreq': testSignals[8:10],
+                            'testJordiNL': [testSignals[0]],
+                            'testWeigth': [testSignals[0]],
+                            'testLPFilterMeanSpectrum': testSignals[10:10+13],
+                            'testBPFilterMeanSpectrum': [testSignals[10]] + list(testSignals[23:23+12]),
+                            'testLPFilterFrameSpeaks': testSignals[10:10+13],
+                            'testBPFilterFrameSpeaks': [testSignals[10]] + list(testSignals[23:23+12]),
+                            'testNormalizeWindow': [testSignals[0]],
+                            'testNormalizeHPCP': [testSignals[0]],
+                           }
+
+# General parameters for signals creation
+fs = 44100.
+t = 3 * 60.  # 3 minutes
+N = t * fs
+f0 = 440.
+sampling = np.arange(int(N))
+semitone = np.power(2.,1./12.)
+f0 = 440.  # A4 => 440 Hz
+precision = 0.0001
+
+# Mapping between tests and  the tones of the signal
+# The tones are expressed as de distance in semitones to the central A 440
+note_scales = {
+        'Tones A0 - G9': [- 5 * 12 , -1 + 4 * 12], 
+        'Tones A0 - G1': [- 5 * 12 , -1 - 4 * 12], 
+        'Tones A1 - G2': [- 4 * 12 , -1 - 3 * 12], 
+        'Tones A2 - G3': [- 3 * 12 , -1 - 2 * 12], 
+        'Tones A3 - G4': [- 2 * 12 , -1 - 1 * 12], 
+        'Tones A4 - G5': [- 1 * 12 , -1 - 0 * 12], 
+        'Tones A5 - G6': [- 0 * 12 , -1 + 1 * 12], 
+        'Tones A6 - G7': [+ 1 * 12 , -1 + 2 * 12], 
+        'Tones A7 - G8': [+ 2 * 12 , -1 + 3 * 12],
+        'Tones A8 - G9': [+ 3 * 12 , -1 + 4 * 12]
+}
+
+
+# Default common parameters for testing
+hpcpSize = 3 * 12
+win_normalized = False
+frameSize_ = 4096
+hopSize_ = frameSize_ / 2    
+speaks_max = 60
+maxFrequencyList = 12000
+JordiNonLinear = False
```

Comment:
This HPCP algorithm's parameter name is now "nonLinear", we should use it here instead.